setup.py

```diff
@@ -13,14 +13,14 @@
     keywords='gis jsonschema',
     packages=find_packages(exclude=['tests']),
     dependency_links=[
-        'https://github.com/azavea/djsonb/tarball/develop#egg=djsonb-0.1.5'
+        'https://github.com/azavea/djsonb/tarball/feature/pattern-search-OR#egg=djsonb-0.1.6'
```

Comment:
My bad on there - had to specify an alternative for provisioning with the in-progress branch to work